src/main/java/com/epam/ta/reportportal/core/analyzer/pattern/impl/CreatePatternTemplateHandlerImpl.java

```diff
@@ -24,31 +23,28 @@
 import com.epam.ta.reportportal.ws.model.ErrorType;
 import com.epam.ta.reportportal.ws.model.project.config.pattern.CreatePatternTemplateRQ;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Service;
 
 import static com.epam.ta.reportportal.commons.Predicates.equalTo;
 
 /**
- * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ * @author <a href="mailto:pavel_bortnik@epam.com">Pavel Bortnik</a>
  */
-@Service
-public class CreateStringPatternTemplateHandler implements CreatePatternTemplateHandler {
+public class CreatePatternTemplateHandlerImpl implements CreatePatternTemplateHandler {
 
-	private final PatternTemplateRepository patternTemplateRepository;
+	protected final PatternTemplateRepository patternTemplateRepository;
 
 	@Autowired
-	public CreateStringPatternTemplateHandler(PatternTemplateRepository patternTemplateRepository) {
+	public CreatePatternTemplateHandlerImpl(PatternTemplateRepository patternTemplateRepository) {
 		this.patternTemplateRepository = patternTemplateRepository;
 	}
 
 	@Override
 	public PatternTemplate createPatternTemplate(Long projectId, CreatePatternTemplateRQ createPatternTemplateRQ) {
-
-		BusinessRule.expect(patternTemplateRepository.existsByProjectIdAndNameIgnoreCase(projectId, createPatternTemplateRQ.getName()),
-				equalTo(false)
-		).verify(ErrorType.RESOURCE_ALREADY_EXISTS, createPatternTemplateRQ.getName());
-
+		final String name = createPatternTemplateRQ.getName().trim();
```

Comment:
May be better use something like 
final String name = StringUtils.trim(createPatternTemplateRQ.getName());
To avoid possible NPE.

Reply:
done